modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="odc-creating-a-binding-connection-between-components_{context}"]
+== Creating a binding connection between components
+
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
+The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+
+.Prerequisites
+
+* You have created and deployed a Spring PetClinic sample application that uses the *Developer* perspective.
+* You have installed the {servicebinding-title} from the OperatorHub.
+* You have installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub by using the `v5` *Update* channel.
```

Comment:
@smunje1 here you are talking about the crunchy postgres operator and above you are asking the user to install redhat postgresql operator. Use the same operator at both places.

Reply:
Hello @divyanshiGupta 
Could we discuss your comment as part of Jira: https://issues.redhat.com/browse/RHDEVDOCS-4234.  For this PR, we are focusing currently only on the RHDEVDOCS-3910 and RHDEVDOCS-3902 Jiras.  The "Creating a binding connection between components" section is out of scope for these Jira's. 

If there are no further comments for sections, "Discovering and identifying Operator-backed bindable services" for RHDEVDOCS-3902 and "Verifying the status of your service binding from the Topology view" for RHDEVDOCS-3910, could you please give ack on this PR?

Thank you!